docs/cpp/configure-intellisense-crosscompilation.md

```diff
@@ -0,0 +1,104 @@
+
+![Intellisense configuration window](images/wsl/intellisense-configurations-wsl.png)
+
+Values in `c_cpp_properties.json` take precedence over the `c_cpp_properties` set in the `settings.json` file.  If there are `c_cpp_properties` set in `settings.json` and the same properties are not set or does not exist in a `c_cpp_properties.json` file, then the extension will use the values in `settings.json`.
+
+If you are curious, you can open the `.vscode/c_cpp_properties.json` file directly. It'll look something like this:
+
+```json
+{
+    "configurations": [
+        {
+            "name": "Linux",
+            "includePath": [
+                "${workspaceFolder}/**"
+            ],
+            "defines": [],
+            "compilerPath": "/usr/bin/gcc",
+            "cStandard": "c11",
+            "cppStandard": "c++17",
+            "IntelliSenseMode": "clang-x64"
+        }
+    ],
+    "version": 4
+}
+```
+
+You can also edit the file directly by opening it from the .vscode subdirectory in your working directory, or by selecting **C/C++: Edit Configurations (JSON)** from the Command Palette (`kb(workbench.action.showCommands)`).
+
+## Set cross-compilation configuration settings
+
+The following settings need to be configured for intellisense to work when you are cross-compiling:
+
+**IntelliSense mode**
+
+![IntelliSense mode setting](images/intellisense/intellisense-mode.png)
+
+Set to the architecture-specific variant of the compiler you are using.
+
+**Compiler path**
+![Compiler path setting](images/intellisense/compiler-path.png)
+
+Set to the full path of the compiler you are using to build your project.
+For example, you might have this path to the Microsoft Visual C++ (MSVC) compiler: `C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Tools/MSVC/14.25.28610/bin/Hostx64/x64/cl.exe`
+
+**Include path**
+
+![Include path setting](images/intellisense/include-path.png)
+
+You only need to modify the **Include path** if your program includes header files that are not in your workspace, or that are in the standard library path.
+
+The C/C++ extension tries to populate the include path by querying the compiler specified by **Compiler path**. If it is unable to get the path for the target system libraries, enter the include path manually. You may need to enter it manually if you are targeting a specific version of the Android NDK if the folder structure under the NDK root folder changes per Android NDK version.
+
+If you are targeting Windows, enter the paths to the Windows SDK and the MSVC libraries.
+
+If the system include path can be retrieved from the compiler, it is sufficient to set `compilerPath` to the compiler that corresponds to the  `IntelliSenseMode`.
+
+If you don't want the extension to query the `compilerPath` to determine system includes, set `"compilerPath": ""`.
+
+## Reusing your C++ configuration
```

Comment:
remove section "Reusing your C++ configuration". This can be part of the tutorial about full customization of IntelliSense settings.

Reply:
done